---
layout: default
title: Research Interests
---

# Research Interests
---------------------

### Bioinformatics tools/algorithms development

As a bioinformatician, I have developed
many useful tools to analyze, visualize, and translate immense biomedical data into knowledge.
These tools have been widely used by academic institutions and industries, and have been
cited hundreds of times. For example:

- I developed the [RSeQC](http://rseqc.sourceforge.net) package
to comprehensively evaluate different aspects of RNA-Seq experiments. RSeQC has been
downloaded tens of thousands of time by people all over the world.  It has been installed
in institute level computing clusters of many institutions including [NIH](https://hpc.nih.gov/apps/rseqc.html),
Boston University, Pennsylvania State University (Galaxy platform), etc.

-  I developed [CPAT](http://lilab.research.bcm.edu/cpat) - to quickly recognize noncoding
RNAs from protein coding mRNAs. CPAT uses a logistic regression model built with four
sequence features. CPAT is not only accurate (sensitivity: 0.96, specificity: 0.97),
but also much more efficient (~10,000 times faster) than other tools, which enables the
users to process thousands of transcripts within seconds. We also developed a web interface
for CPAT that allows users to submit sequences and receive the prediction results almost
instantly.

- I developed [CrossMap](http://crossmap.sourceforge.net/), a versatile and efficient
tool for converting genome analysis results between assemblies. It has been used by
[Ensembl](http://www.ensembl.org/) since 2015 to lift over genomic analysis results, it
is also incorporated into Illumina [BaseSpace cloud computing platform](https://basespace.illumina.com/home).
 

### Biomarker discovery

Cancer cells are heterogeneous and no cancer treatment is 100% effective against disseminated cancer.
Drug resistance results from a variety of factors including germline variations in patients and
somatic alterations in tumors. Identification of biomarkers associated with drug resistance
will help design new therapeutic agents, and enable precise medicine.

- Identification of genomic and transcriptomic aberrations associated with primary resistance
to abiraterone acetate (ZYTIGA®) treatments in metastatic castration-resistant prostrate cancer
(mCRPC), we performed whole-exome and RNA sequencing of 86 metastases (and blood) in pre-chemotherapy
mCRPC patients initiating abiraterone acetate/prednisone treatment. We found that genes involved
in the Wnt/β-catenin pathway are frequently mutated, and negative regulators of the Wnt pathway
such as DKK4, SFRP2 and LRP6 were more frequently deleted in patients exhibiting intrinsic resistance
to abiraterone acetate.

- Through integrative analyses of the copy-number profiles of 10,759 patients across 31
cancer types from The Cancer Genome Atlas (TCGA) project, we found the homozygous deletion
of interferon (P = 0.0029, OR = 11.8) and defensin (P = 0.06, OR = 2.79) genes are significantly
associated with resistance to anti-CTLA4 immunotherapy.

- Identification of novel DNA methylation signatures to stratify Alpha-1 Antitrypsin
Deficiency (AATD) liver disease patients. 