---
layout: default
title: Bioinformatics Tools
subtitle: Bioinformatics Tools
---

## Bioinformatics Tools developed by our team

### [RSeQC](http://rseqc.sourceforge.net)

- RSeQC package provides many useful python modules to QC, analyze and visualize RNA-seq data.
- Reference: Wang L, Wang S, Li W. [RSeQC: quality control of RNA-seq experiments](https://pubmed.ncbi.nlm.nih.gov/22743226/). **Bioinformatics**. 2012 Aug 15;28(16):2184-5. doi: 10.1093/bioinformatics/bts356. Epub 2012 Jun 27. PMID: 22743226.

### [CPAT](http://rna-cpat.sourceforge.net/)

- CPAT (Coding-Potential Assessment Tool) trained a logistic regression model from pure sequence features to distinguish noncoding RNAs from protein coding mRNAs.
- [Webserver](http://lilab.research.bcm.edu/cpat)   
- Reference: Wang L, Park HJ, Dasari S, Wang S, Kocher JP, Li W. [CPAT: Coding-Potential Assessment Tool using an alignment-free logistic regression model](https://pubmed.ncbi.nlm.nih.gov/23335781/). **Nucleic Acids Res**. 2013 Apr 1;41(6):e74. doi: 10.1093/nar/gkt006. Epub 2013 Jan 17. PMID: 23335781; PMCID: PMC3616698.

### [CrossMap](http://crossmap.sourceforge.net)

- CrossMap is a program to liftover genome coordinates between different genome assemblies. It supports multiple file formats including BED, BAM/SAM/CRAM, VCF, Wiggle/BigWig, etc. 
- Reference: Zhao H, Sun Z, Wang J, Huang H, Kocher JP, Wang L. [CrossMap: a versatile tool for coordinate conversion between genome assemblies](https://pubmed.ncbi.nlm.nih.gov/24351709/). **Bioinformatics**. 2014 Apr 1;30(7):1006-7. doi: 10.1093/bioinformatics/btt730. Epub 2013 Dec 18. PMID: 24351709; PMCID: PMC3967108.

### [MACE](http://chipexo.sourceforge.net/)

- MACE (Model-based Analysis of ChIP-Exo) is a program dedicated to analyse [ChIP-exo](https://en.wikipedia.org/wiki/ChIP-exo) data. 
- Reference: Wang L, Chen J, Wang C, Uusk√ºla-Reimand L, Chen K, Medina-Rivera A, Young EJ, Zimmermann MT, Yan H, Sun Z, Zhang Y, Wu ST, Huang H, Wilson MD, Kocher JP, Li W. [MACE: model based analysis of ChIP-exo](https://pubmed.ncbi.nlm.nih.gov/25249628/). **Nucleic Acids Res**. 2014 Nov 10;42(20):e156. doi: 10.1093/nar/gku846. Epub 2014 Sep 23. PMID: 25249628; PMCID: PMC4227761.


### [Epidaurus](http://epidaurus.sourceforge.net/)

- Epidaurus is software to analyze and visualize different types of epigenomic data. 
- Reference: Wang L, Huang H, Dougherty G, Zhao Y, Hossain A, Kocher JP. [Epidaurus: aggregation and integration analysis of prostate cancer epigenome](https://pubmed.ncbi.nlm.nih.gov/25378314/). **Nucleic Acids Res**. 2015 Jan;43(2):e7. doi: 10.1093/nar/gku1079. Epub 2014 Nov 5. PMID: 25378314; PMCID: PMC4333365.


### [PVAAS](http://pvaas.sourceforge.net/)

- PVAAS is a program to identify single nucleotide variants that are associated with aberrant RNA splicing. 
- Reference: Wang L, Nie JJ, Kocher JP. [PVAAS: identify variants associated with aberrant splicing from RNA-seq](https://pubmed.ncbi.nlm.nih.gov/25573917/). **Bioinformatics**. 2015 May 15;31(10):1668-70. doi: 10.1093/bioinformatics/btv001. Epub 2015 Jan 7. PMID: 25573917; PMCID: PMC4807355.

### [CircularLogo](https://sourceforge.net/projects/circularlogo)

- CircularLogo is an innovative web application that specifically designed to visualize and explore intra-motif dependencies. It is able to display intra-motif dependencies and reveal biomolecular structure effectively. 
- Reference: Ye Z, Ma T, Kalmbach MT, Dasari S, Kocher JA, Wang L. [CircularLogo: A lightweight web application to visualize intra-motif dependencies](https://pubmed.ncbi.nlm.nih.gov/28532394/). **BMC Bioinformatics**. 2017 May 22;18(1):269. doi: 10.1186/s12859-017-1680-2. PMID: 28532394; PMCID: PMC5440937.

### [TIN](http://rseqc.sourceforge.net)
- TIN (Transcript Integrity Number) is an algorithm to measure RNA degradation at transcript level. It can improve differential gene expression analyses by reducing false positives and false negatives.
- Reference: Wang L, Nie J, Sicotte H, Li Y, Eckel-Passow JE, Dasari S, Vedell PT, Barman P, Wang L, Weinshiboum R, Jen J, Huang H, Kohli M, Kocher JP. [Measure transcript integrity using RNA-seq data](https://pubmed.ncbi.nlm.nih.gov/26842848/). **BMC Bioinformatics**. 2016 Feb 3;17:58. doi: 10.1186/s12859-016-0922-z. PMID: 26842848; PMCID: PMC4739097.

### [CpGtools](https://cpgtools.readthedocs.io/en/latest/)

- CpGtools is a Python package to perform DNA methylation analysis.
- Reference: Wei T, Nie J, Larson NB, Ye Z, Eckel-Passow JE, Robertson KD, Kocher JA, Wang L. [CpGtools: a python package for DNA methylation analysis](https://pubmed.ncbi.nlm.nih.gov/31808791/). **Bioinformatics**. 2021 Jul 12;37(11):1598-1599. doi: 10.1093/bioinformatics/btz916. PMID: 31808791; PMCID: PMC8275978.

### [Spiker](https://spiker.readthedocs.io/en/latest/)
- Spiker is a Python package to analzye Spike-In ChIP-seq data.
- Reference: Wu D, Wang L, Huang H. [Protocol to apply spike-in ChIP-seq to capture massive histone acetylation in human cells](https://pubmed.ncbi.nlm.nih.gov/34337446/). **STAR Protoc**. 2021 Jul 17;2(3):100681. doi: 10.1016/j.xpro.2021.100681. PMID: 34337446; PMCID: PMC8313745.

### [cobind](https://cobind.readthedocs.io/en/latest/)
- Cobind is designed for quantitative analysis of the genomic overlaps.
- Reference: Ma T, Guo L, Yan H, and Wang L. Cobind: quantitative analysis of the genomic overlaps. **Bioinformatics Advances** (accepted).
