---
layout: page
title: Bioinformatics Tools
subtitle: " "
---

### [RSeQC](http://rseqc.sourceforge.net)

- Description: RSeQC package provides many useful python modules to QC, analyze and visualize RNA-seq data.
- Website: http://rseqc.sourceforge.net
- Reference: Wang, L., Wang, S., & Li, W. (2012). RSeQC: quality control of RNA-seq experiments. Bioinformatics (Oxford, England), 28(16), 2184–2185. http://doi.org/10.1093/bioinformatics/bts356

### [CPAT] (http://rna-cpat.sourceforge.net/)

- Description: CPAT (Coding-Potential Assessment Tool) uses the logistic regression model to distinguish noncoding RNAs from protein coding mRNAs.
- Website: http://rna-cpat.sourceforge.net/ 
- Webserver: http://lilab.research.bcm.edu/cpat/   
- Reference: Wang, L., Park, H. J., Dasari, S., Wang, S., Kocher, J.-P., & Li, W. (2013). CPAT: Coding-Potential Assessment Tool using an alignment-free logistic regression model. Nucleic Acids Research, 41(6), e74. http://doi.org/10.1093/nar/gkt006

### [CrossMap](http://crossmap.sourceforge.net)

- Description: CrossMap is a program to liftover genome coordinates between different genome assemblies.
- Website: http://crossmap.sourceforge.net/ 
- Reference: Reference: Zhao, H., Sun, Z., Wang, J., Huang, H., Kocher, J.-P., & Wang, L. (2014). CrossMap: a versatile tool for coordinate conversion between genome assemblies. Bioinformatics (Oxford, England), 30(7), 1006–1007. http://doi.org/10.1093/bioinformatics/btt730

### [MACE] (http://chipexo.sourceforge.net/)

- Description: MACE (Model-based Analysis of ChIP-Exo) is a program dedicated to analyse ChIP-exo data. 
- Website: http://chipexo.sourceforge.net/ 
- Reference: Wang, L., Chen, J., Wang, C., Uusküla-Reimand, L., Chen, K., Medina-Rivera, A., et al. (2014). MACE: model based analysis of ChIP-exo. Nucleic Acids Research. http://doi.org/10.1093/nar/gku846


### [Epidaurus] (http://epidaurus.sourceforge.net/)

- Description: Epidaurus is software to perform integrative analysis of different types of epigenomic data. 
- Website: http://epidaurus.sourceforge.net/ 
- Reference: Wang, L., Huang, H., Dougherty, G., Zhao, Y., Hossain, A., & Kocher, J.-P. A. (2015). Epidaurus: aggregation and integration analysis of prostate cancer epigenome. Nucleic Acids Research, 43(2), e7. http://doi.org/10.1093/nar/gku1079


### [PVAAS] (http://pvaas.sourceforge.net/)

- Description: PVAAS is a program to identify single nucleotide variants that are associated with aberrant splicing. 
- Website: http://pvaas.sourceforge.net/ 
- Reference: Wang, L., Nie, J. J., & Kocher, J.-P. A. (2015). PVAAS: identify variants associated with aberrant splicing from RNA-seq. Bioinformatics (Oxford, England), 31(10), 1668–1670. http://doi.org/10.1093/bioinformatics/btv001

### [CircularLogo](https://sourceforge.net/projects/circularl)

- Description: CircularLogo is an innovative web application that specifically designed to visualize and explore intra-motif dependencies. It is able to display intra-motif dependencies and reveal biomolecular structure effectively. 
- Website: https://sourceforge.net/projects/circularl
- Reference: Ye, Z., Ma, T., Kalmbach, M. T., Dasari, S., Kocher, J.-P. A., & Wang, L. (2017). CircularLogo: A lightweight web application to visualize intra-motif dependencies. BMC Bioinformatics, 18(1), 269. http://doi.org/10.1186/s12859-017-1680-2 (Corresponding author)

### [TIN] (http://rseqc.sourceforge.net)
- Description: TIN (Transcript Integrity Number) is an algorithm to measure RNA degradation at transcript level. It significantly neutralizes RNA degradation effects and improves gene expression analyses. It has broad application in clinical RNA-seq data analysis
- Website: http://rseqc.sourceforge.net/
- Reference: Wang, L., Nie, J., Sicotte, H., Li, Y., Eckel-Passow, J. E., Dasari, S., et al. (2016). Measure transcript integrity using RNA-seq data. BMC Bioinformatics, 17(1), 1–16. http://doi.org/10.1186/s12859-016-0922-z

### [CpGtools] (https://cpgtools.readthedocs.io/en/latest/)

- Description: CpGtools is a Python package to perform DNA methylation analysis. 
- Website: https://github.com/liguowang/cpgtools 
- Documentation: https://cpgtools.readthedocs.io/en/latest/ 
- Reference: Wei T, Nie J, Larson NB, Ye Z, Eckel Passow JE, Robertson KD, Kocher JA, Wang L. CpGtools: A Python Package for DNA Methylation Analysis. Bioinformatics. 2019 Dec 6 Epub 2019 Dec 06

