---
layout: default
title: Bioinformatics Tools
subtitle: Bioinformatics Tools
---

## Bioinformatics Tools developed by our team

### [RSeQC](http://rseqc.sourceforge.net)

- RSeQC package provides many useful python modules to QC, analyze and visualize RNA-seq data.
- [Documentation](http://rseqc.sourceforge.net)
- [Paper](https://academic.oup.com/bioinformatics/article/28/16/2184/325191)

### [CPAT](http://rna-cpat.sourceforge.net/)

- CPAT (Coding-Potential Assessment Tool) trained a logistic regression model from pure sequence features to distinguish noncoding RNAs from protein coding mRNAs.
- [Documentation](http://rna-cpat.sourceforge.net/) 
- [Webserver](http://lilab.research.bcm.edu/cpat)   
- [Paper](https://academic.oup.com/nar/article/41/6/e74/2902455)

### [CrossMap](http://crossmap.sourceforge.net)

- CrossMap is a program to liftover genome coordinates between different genome assemblies. It supports multiple file formats including BED, BAM/SAM/CRAM, VCF, Wiggle/BigWig, etc. 
- [Documentation](http://crossmap.sourceforge.net/)
- [Paper](https://academic.oup.com/bioinformatics/article/30/7/1006/234947)

### [MACE](http://chipexo.sourceforge.net/)

- MACE (Model-based Analysis of ChIP-Exo) is a program dedicated to analyse [ChIP-exo](https://en.wikipedia.org/wiki/ChIP-exo) data. 
- [Documentation](http://chipexo.sourceforge.net/)
- [Paper](https://academic.oup.com/nar/article/42/20/e156/2902729)


### [Epidaurus](http://epidaurus.sourceforge.net/)

- Epidaurus is software to analyze and visualize different types of epigenomic data. 
- [Documentation](http://epidaurus.sourceforge.net/)
- [Paper](https://academic.oup.com/nar/article/43/2/e7/2414228)


### [PVAAS](http://pvaas.sourceforge.net/)

- PVAAS is a program to identify single nucleotide variants that are associated with aberrant RNA splicing. 
- [Documentation](http://pvaas.sourceforge.net)
- [Paper](https://academic.oup.com/bioinformatics/article/31/10/1668/176935)

### [CircularLogo](https://sourceforge.net/projects/circularlogo)

- CircularLogo is an innovative web application that specifically designed to visualize and explore intra-motif dependencies. It is able to display intra-motif dependencies and reveal biomolecular structure effectively. 
- [Documentation](https://sourceforge.net/projects/circularlogo)
- [Paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1680-2)

### [TIN](http://rseqc.sourceforge.net)
- TIN (Transcript Integrity Number) is an algorithm to measure RNA degradation at transcript level. It can improve differential gene expression analyses by reducing false positives and false negatives.
- [Documentation](http://rseqc.sourceforge.net)
- [Paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0922-z)

### [CpGtools](https://cpgtools.readthedocs.io/en/latest/)

- CpGtools is a Python package to perform DNA methylation analysis. 
- [Website](https://github.com/liguowang/cpgtools)
- [Documentation](https://cpgtools.readthedocs.io/en/latest) 
- [Paper](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btz916/5663456)

### [Spiker](https://spiker.readthedocs.io/en/latest/) is a Python package to analzye Spike-In ChIP-seq data.
- [Documentation](https://spiker.readthedocs.io/en/latest/)
- [Paper](https://pubmed.ncbi.nlm.nih.gov/34337446/) 
